.changeset/silly-clocks-tan.md

```diff
@@ -18,21 +18,41 @@ Users desiring the previous strict behavior should use `view.compatibility.isEqu
 # `ITree.schematize` deprecation
 
 `ITree.schematize` has been deprecated in favor of `ITree.viewWith`.
-Unlike `schematize`, `viewWith` does not implicitly initialize the document. As such, it doesn't take an `initialTree` property.
+Unlike `schematize`, `viewWith` does not implicitly initialize the document.
+As such, it doesn't take an `initialTree` property.
 Instead, applications should initialize their trees in document creation codepaths using the added `TreeView.initialize` API.
 
+## Old
+
 As an example, something like the following code may have been used before for both the document create and document load codepaths:
 
 ```typescript
+// -- first-party API --
 const tree = SharedTree.create(runtime, "foo");
 const view = tree.schematize(new TreeConfiguration(Point, () => new Point({ x: 0, y: 0 })));
 ```
 
-Now, that code would look like this on the create codepath:
+When using the third-party API, creating a tree looks a bit different but the call to `schematize` is the same:
+
+```typescript
+// -- third-party API for statically defined objects in container schema --
```

Comment:
Given that we prefer first parties to use this API if practical, and third parties can technically do whatever they want (we don't actually have any special first party access), I don't think thats a great way to describe this API surface. I think any of `fluid-framework`, "declarative", `ContainerSchema`, "Service Client" or `IFluidContainer`.

Also I'd suggest putting this API first as its the public one we want people to use.

What you have called the "first party API" is called the `encapsulated API`, but its also exported under /legacy, you calling it the "legacy" or "/legacy" API would also make sense ( https://fluidframework.com/docs/build/releases-and-apitags/calls it legacy, though is incorrect about how its tagged).

https://github.com/microsoft/FluidFramework/wiki/Release-Tags uses the term "known partners" not first-party.
I think we should avoid introducing anything that claims to be special first party stuff. For developers outside MS contributing to this repo, first party likely won't mean what I think you are intending here.